usaspending_api/bulk_download/v2/views.py

```diff
@@ -0,0 +1,311 @@
+import sys
+import itertools
+import json
+import time
+import logging
+
+from django.conf import settings
+from django.db.models import F, Sum, Value, CharField, Q
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework.exceptions import NotFound, ParseError
+
+from usaspending_api.awards.v2.lookups.lookups import contract_type_mapping, \
+    grant_type_mapping, direct_payment_type_mapping, loan_type_mapping, other_type_mapping
+from usaspending_api.awards.models import Award, Subaward, TransactionNormalized, Agency
+from usaspending_api.references.models import ToptierAgency, SubtierAgency
+from usaspending_api.accounts.models import FederalAccount
+from usaspending_api.common.exceptions import InvalidParameterException
+from usaspending_api.common.csv_helpers import sqs_queue
+from usaspending_api.bulk_download.filestreaming import csv_selection
+from usaspending_api.bulk_download.filestreaming.s3_handler import S3Handler
+from usaspending_api.bulk_download.models import BulkDownloadJob
+from usaspending_api.download.lookups import JOB_STATUS_DICT
+
+logger = logging.getLogger('console')
```

Comment:
Should this be changed to a log file?

Reply:
I think the settings automatically makes a log file for everything logged in the console. See lines 214-273 of `settings.py`.